### DISCUSSION TITLE
list installed AUR package only

### DISCUSSION THREAD

**Discussion by realyukii** [Morganamilo/paru#1216] [Q&A] [answered]
Is Paru able to print the list of installed AUR- packages only?

**Comment by Morganamilo** [comment, marked as answer]
`pacman -Qmq` lists foreign packages which is usually the same thing. But any packages that are not in the repos (i.e. pkgbuilds you've written yourself) will also show up there.

You can filter that list through AUR packages only if you need to with `pacman -Qnq | paru -Ta -`.

**Reply by ShaiAvr** [reply]
Did you mean `pacman -Qmq | paru -Ta -`? The command you provided returns an empty list for me, and `-m` does work. Also, both commands exit with 127 code. Is this normal?
